pgrx-macros/src/rewriter.rs

```diff
@@ -135,19 +136,52 @@ fn foreign_item_fn(func: &ForeignItemFn, abi: &syn::Abi) -> syn::Result<proc_mac
     let arg_list = rename_arg_list(&func.sig)?;
     let arg_list_with_types = rename_arg_list_with_types(&func.sig)?;
     let return_type = func.sig.output.clone();
+    let link_with_cshim = func.attrs.iter().any(|attr| match &attr.meta {
+        syn::Meta::NameValue(kv) => {
```

Comment:
the `if` should be able to be combined into the match arm, something like:
```suggestion
        syn::Meta::NameValue(kv) if kv.path.get_ident().filter(|x| x == "link_name").is_some() => {
```